Zend/zend_portability.h

```diff
@@ -223,9 +223,11 @@ char *alloca();
 #if ZEND_GCC_VERSION >= 4003 || __has_attribute(alloc_size)
```

Comment:
This seems completely unrelated to `__has_attribute(alloc_size)`, no? We should probably move it to it's own block.

Reply:
works for me